### PR TITLE
support desired formation for 3 cables

### PR DESCRIPTION
**Pull request by KhWahba**

### CODE REVIEW

File: src/modules/src/controller_lee_payload.c [hunk 1]
```diff
@@ -1565,6 +1559,28 @@ static void runQP(const struct QPInput *input, struct QPOutput* output)
...
+          } else {
+            float scale = vmag(F_d) / 3.0f;
+            if (vmag2(muPlanned) > 0) {
+              muPlanned = vsclnorm(muPlanned, scale);
+            }
+            if (vmag2(muPlanned2) > 0) {
+              muPlanned2 = vsclnorm(muPlanned2, scale);
+            }
+            if (vmag2(muPlanned3) > 0) {
+              muPlanned2 = vsclnorm(muPlanned3, scale);
```
Comment (whoenig) [inline src/modules/src/controller_lee_payload.c]: typo...

File: src/modules/src/controller_lee_payload.c [hunk 2]
```diff
@@ -1807,23 +1823,36 @@ void controllerLeePayload(controllerLeePayload_t* self, control_t *control, setp
...
     // Set corresponding desired cable angles points
-    DEBUG_PRINT("%d\n", setpoint->num_cables);
-
+    static int counter = 0;
+    if (counter % 100 == 0) {
+      DEBUG_PRINT("formation control: %d\n", self->formation_control);
+      DEBUG_PRINT("cable nums %d\n", setpoint->num_cables);
+      DEBUG_PRINT("num of neighbors: %d\n", state->num_neighbors);
+    }
+    counter++;
```
Comment (whoenig) [inline src/modules/src/controller_lee_payload.c]: Do we still need that?

**Review by whoenig — CHANGES_REQUESTED** [review]

File: src/modules/src/controller_lee_payload.c (hunk 2 above)
Comment (KhWahba) [reply src/modules/src/controller_lee_payload.c]: No, I will remove it

File: src/modules/src/controller_lee_payload.c (hunk 1 above)
Comment (KhWahba) [reply src/modules/src/controller_lee_payload.c]: Done

**Pull request merged**